Header header
int32[] map
geometry_msgs/Point32 origin
geometry_msgs/Vector3 resolutions
int16 size_x
int16 size_y
int16 size_z
